index.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  extends: ['plugin:react/recommended', 'plugin:prettier/recommended'],
+  plugins: ['import', 'unicorn', 'prettier'],
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  globals: {
+    require: true,
+    process: true,
+  },
+  rules: {
+    'default-case': 'error',
+    'import/extensions': [
+      'error',
+      'always',
+      { js: 'never', jsx: 'never', ts: 'never', tsx: 'never', hbs: 'never', vue: 'never' },
+    ],
+    'import/no-unresolved': 'error',
+    'max-lines': 'error',
+    'no-console': 'error',
+    'no-debugger': 'error',
+    'no-extra-boolean-cast': 'off',
+    'no-restricted-properties': [
+      'error',
+      { object: 'document', property: 'querySelector' },
+      { object: 'document', property: 'querySelectorAll' },
+    ],
+    'no-undef': 'error',
+    'no-underscore-dangle': 'error',
+    'prettier/prettier': [
```

Comment:
Extending 'plugin:prettier/recommended' already does everything for you so there's no need to explicitly set prettier rules as error. (https://prettier.io/docs/en/integrating-with-linters.html)